modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/workflow/processors/SW360Enricher.java

```diff
@@ -17,10 +17,17 @@
 import org.eclipse.sw360.antenna.sw360.SW360MetaDataReceiver;
 import org.eclipse.sw360.antenna.sw360.workflow.SW360ConnectionConfiguration;
 
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.Collection;
 import java.util.Map;
 
 public class SW360Enricher extends AbstractProcessor {
+    private static final String DOWNLOAD_ATTACHMENTS = "download.attachments";
+    private static final String DOWNLOAD_ATTACHMENTS_DIR="download.directory";
+    private boolean downloadAttachments;
+    private Path downloadPath = Paths.get("");
```

Comment:
Probably we could make the default downloadPath to antenna target directory?

Reply:
This would result in an NPE, since at the time of instantiation the context is null for the enricher

Reply:
Yes, you can do it in `configure`.